pymathics/graph/base.py

```diff
@@ -14,7 +14,7 @@
 from typing import Callable, Optional, Tuple
```

Comment:
Tuple is no longer used. Nor is `base64` or `tempfile` on lines above.  Running this over flake8 will find other things.